v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -7,28 +7,118 @@ category: advanced
 keywords:
 ---
 
-SS58 is a simple address format designed for Substrate based chains. There's no problem with using
-other address formats for a chain, but this serves as a robust default. It is heavily based on
-Bitcoin's Base-58-check format with a few alterations.
+SS58 is a simple address format designed for Substrate based chains.
+There's no problem with using other address formats for a chain, but this serves as a robust default.
+It is heavily based on Bitcoin's Base-58-check format with a few alterations.
 
-The basic idea is a base-58 encoded value that can identify a specific account on the Substrate
-chain. Different chains have different means of identifying accounts. SS58 is designed to be
-extensible for this reason.
+The basic idea is a base-58 encoded value which can identify a specific account on the Substrate chain.
+Different chains have different means of identifying accounts.
+SS58 is designed to be extensible for this reason.
+
+## Format in detail
+
+This page outlines the implementation of [`Ss58Codec` in Substrate](https://paritytech.github.io/substrate/master/sp_core/crypto/trait.Ss58Codec.html).
+Also of note is the cannonical [SS58 registry](https://github.com/paritytech/ss58-registry) of adress type mappings to various networks described below.
+
+### Basic format
+
+The basic format conforms to:
+
+```
+base58encode ( concat ( <address-type>, <address>, <checksum> ) )
+```
+
+That is, the concatenated byte series of address type, address and checksum then passed into a base-58 encoder.
+The `base58encode` function is implemented exactly as [defined](https://en.wikipedia.org/wiki/Base58) in Bitcoin and IPFS, using the same alphabet as both.
+
+### Address type
+
+The `<address-type>` is one or more bytes that describe the precise format of the following bytes.
+
+Currently, there exist several valid values:
+
+- 00000000b..=00111111b (0..=63 inclusive): Simple account/address/network identifier.
+  The byte can be interpreted directly as such an identifier.
+- 01000000b..=01111111b (64..=127 inclusive): Full address/address/network identifier.
+  The lower 6 bits of this byte should be treated as the upper 6 bits of a 14 bit identifier value, with the lower 8 bits defined by the following byte. 
+  This works for all identifiers up to 2**14 (16,383).
+- 10000000b..=11111111b (128..=255 inclusive): Reserved for future address format extensions.
+
+The latter (42) is a "wildcard" address that is meant to be equally valid on all Substrate networks that support fixed-length addresses.
+For production networks, however, a network-specific version may be desirable to help avoid the key-reuse between networks and some of the problems that it can cause.
+Substrate nodes will default to printing keys in address type 42, though alternative Substrate-based node implementations (e.g. Polkadot) may elect to default to some other type.
+
+### Address Formats for Substrate
```

Comment:
Is the below table wrong? I see https://paritytech.github.io/substrate/master/src/sp_core/crypto.rs.html#245 has 2 bytes for _all_ formats...?